common/tor.c

```diff
@@ -17,8 +17,7 @@
 #include <unistd.h>
 #include <wire/wire.h>
 
-
-bool parse_tor_wireaddr(const char *arg,u8 *ip_ld,u16 *port_ld)
+bool parse_tor_wireaddr(const char *arg, u8 * ip_ld, u16 * port_ld)
```

Comment:
I want to ask again... why cannot use `parse_wireaddr` for this?

Reply:
at first i thought about some special cases but maybe a hidden proxy but you right at the end all fits easy just in normal wireaddr . And makes things more uniform addr is addr peer makes it a node so i will put it to wireaddr and replace the inner with call to parse_wireaddr.
and rename parse_tor_wireaddr handle_proxy_option ok? :-)